lib/guardian/plug/ensure_permissions.ex

```diff
@@ -36,7 +42,8 @@ defmodule Guardian.Plug.EnsurePermissions do
     on_failure = Map.get(opts, :on_failure)
     key = Map.get(opts, :key, :default)
     handler = Map.get(opts, :handler)
-    perms = Map.drop(opts, [:handler, :on_failure, :key])
+    one_of = Map.get(opts, :one_of)
+    perms = Map.drop(opts, [:handler, :on_failure, :key, :one_of])
```

Comment:
I think this one would be better written to have the same shape. Something like

```elixir
single_set = Map.drop(opts, [:handler, :on_failure, :key, :one_of])
perm_sets = Map.get(opts, :one_of, [single_set])
```
Then the keys down below would be:

```elixir
%{
  handler: handler,
  key: key,
  perm_sets: perm_sets
}
```